plugins/UM3NetworkPrinting/resources/qml/PrinterCardProgressBar.qml

```diff
@@ -0,0 +1,120 @@
+import QtQuick 2.3
```

Comment:
The JS in this file could use some more ;s.

Reply:
Hmm yes that's one which I think I pretty much just copy pasted.

I think tomorrow I'll take an hour to go through all the files and check for missing semicolons.

Reply:
Semi-colons are not needed for JS in QML.